rc_dev

```diff
@@ -21,3 +21,4 @@ export VECTOR_PROFILENAME=chsdi_aws_admin_ro
 export WMSHOST=wms-bgdi.dev.bgdi.ch
 export WSGI_THREADS=15
 export CMSGEOADMINHOST=https://cms.geo.admin.ch
+export LINKEDDATA_HOST=https://ld.geo.admin.ch
```

Comment:
All hosts don't have underscore, or at least keep the convention env variable with caps and app variable without,